app/models/manageiq/providers/amazon/inventory/collector/cloud_manager.rb

```diff
@@ -21,6 +21,10 @@ def cloud_database_flavors
 
   def cloud_databases
     hash_collection.new(aws_rds.client.describe_db_instances.flat_map(&:db_instances))
+  rescue
```

Comment:
NOTE: AWS doesn't have a "base" exception that we could catch here, and there are a number of different exceptions that can be raised in different circumstances (e.g. NetworkingError, AccessDenied, etc...).  These inherit from StandardError only hence no more specific exception being caught here.